app/templates/about.jade

```diff
@@ -372,23 +372,13 @@ block content
                 li.small(data-i18n="about.jobs_benefit_7")
           .col-sm-6.col-md-5.col-md-offset-1.col-lg-4.col-lg-offset-0
             .job-listing
-              h5 Head of Operations (China)
-                .text-center
-                  small.label
-                    | Beijing • Full-time
-                p.small Help us get our China office up and running!
-                p.small CodeCombat is searching for a leader to build our China operations team in Beijing.
-                a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/7c9371ba-c41f-4de2-8deb-0911e65bdd7a?lever-origin=applied&lever-source%5B%5D=CodeCombat" rel="external")
-                  span(data-i18n="about.learn_more")
-          .col-sm-6.col-md-5.col-md-offset-1.col-lg-4.col-lg-offset-0
-            .job-listing
-              h5 Partnership Development Lead (China)
+              h5 Software Engineer (China)
                 .text-center
                   small.label
                     | Beijing • Full-time
                 p.small Want to make a difference on a global scale?
-                p.small CodeCombat is looking for an entrepreneurial leader to head business development in Beijing.
-                a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/31c8ed3e-e415-46db-b9f4-e8472151df9d?lever-origin=applied&lever-source%5B%5D=CodeCombat" rel="external")
+                p.small CodeCombat is looking for an experienced software engineer to support the education platform of the future.
+                a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/623b1bba-5686-4112-86ff-f99add32e8f3" rel="external")
```

Comment:
Does it matter that this doesn't open in a new tab?

If we wanted that you can add:
```suggestion
                a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/623b1bba-5686-4112-86ff-f99add32e8f3" rel="external" target="_blank")
```